### PR TITLE
Update minitft_featherwing.py

### PR DESCRIPTION
**Pull request by hexthat**
Set the SPI speed, I was having some issues with it being way to slow.

### CODE REVIEW

File: adafruit_featherwing/minitft_featherwing.py [hunk 1]
```diff
@@ -74,11 +74,16 @@ def __init__(self, address=0x5E, i2c=None, spi=None):
         self._backlight.duty_cycle = 0
         self._ss.pin_mode_bulk(self._button_mask, self._ss.INPUT_PULLUP)
         displayio.release_displays()
-        display_bus = displayio.FourWire(spi, command=board.D6, chip_select=board.D5)
+        spi.try_lock()
```
Comment (jerryneedell) [inline adafruit_featherwing/minitft_featherwing.py]: I don't think you can assume the try_lock() will be "True" immediately - you need tp test and wait for it to be True - possibly with a timeout and error.

Comment (jerryneedell) [reply adafruit_featherwing/minitft_featherwing.py]: the timeout is probably overkill, but I recommend using 
```
while not spi.try_lock():
    pass
```
to make sure the lock state is True.

Comment (hexthat) [reply adafruit_featherwing/minitft_featherwing.py]: I changed line to what you suggested.

**Comment by jerryneedell** [comment]
@hexthat I don't know if you see the errors from the travis pylint but it wants you to fix the spacing on some continuation lines
```
************* Module adafruit_featherwing.minitft_featherwing
C: 84, 0: Wrong continued indentation (add 1 space).
                                        command=board.D6,
                                        ^| (bad-continuation)
C: 85, 0: Wrong continued indentation (add 1 space).
                                        chip_select=board.D5)
                                        ^| (bad-continuation)
C: 87, 0: Wrong continued indentation (remove 8 spaces).
                                       rotation=270, bgr=True)
                               |       ^ (bad-continuation)
```

**Comment by hexthat** [comment]
okay i fixed that line to what it was

**Review by makermelissa — APPROVED** [review]
Tested and it works great. Fixes #40 as well.

**Pull request merged**